package.json

```diff
@@ -32,16 +32,17 @@
   },
   "scripts": {
     "build": "tsc",
-    "lint": "eslint . --ext .js,.ts",
+    "lint": "npm run lint:types; npm run lint:js; npm run lint:docs",
+    "lint:js": "eslint . --ext .js,.ts",
     "lint:docs": "npx -y alex .",
+    "lint:types": "tsc --noEmit",
     "prebuild": "rm -rf dist/",
     "prepack": "npm run build",
     "prepare": "husky install",
     "pretest": "npm run lint",
     "prettier": "prettier --list-different \"./**/**.{js,ts,md}\"",
```

Comment:
doesn't have to happen in this PR but thoughts on running prettier against everything?

Reply:
i think this is already doing that? we don't have any yaml files in here for testing flows

Reply:
looks like prettier is flagging a few other files:

<img width="466" alt="CleanShot 2023-08-14 at 11 43 58@2x" src="https://github.com/readmeio/oas/assets/8854718/fc0a157c-e9d8-457a-8623-bc22879dcce4">